tools/llvm-spirv/CMakeLists.txt

```
@@ -23,3 +26,10 @@ target_include_directories(llvm-spirv
     ${LLVM_INCLUDE_DIRS}
     ${LLVM_SPIRV_INCLUDE_DIRS}
 )
+
+if(LLVM_SPIRV_LIBSPIRV_DIS)
+  pkg_search_module(SPIRV_TOOLS REQUIRED SPIRV-Tools)
```

May be something like this can work (not sure about the correctness of pkg_search_module semantic)?

```suggestion
  pkg_search_module(SPIRV_TOOLS QUIET SPIRV-Tools SPIRV_TOOLS_FOUND)
  if(SPIRV_TOOLS_FOUND)
    add_definitions(LLVM_SPIRV_LIBSPIRV_DIS)
```

We would end up with two possible builds, but with REQUIRES specifier in the tests we can test this functionality without spawning extra CI jobs.

---

In the Khronos OpenCL Tooling call yesterday we discussed enabling the new cmake option in CI.  But following your suggestion, we can just enable the functionality if SPIRV-Tools is available without the need for a cmake option.  This has the additional benefit that we can hoist the SPIRV-Tools check to the top-level `CMakeLists.txt`, and reuse its result in `test/CMakeLists.txt`.  I'll update the PR for this.